### PR TITLE
[TRANSLATION] Update Chinese translation (zh_CN & zh_TW)

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

File: dll/win32/userenv/userenv.rc [hunk 1]
```diff
@@ -122,4 +122,7 @@ LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
 #endif
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
+#endif
+#ifdef LANGUAGE_ZH_TW
+    #include "lang/zh-TW.rc"
 #endif
```
Comment (katahiromz) [inline dll/win32/userenv/userenv.rc]: Add newline at end of file.

File: modules/rosapps/applications/vcdcontroltool/lang/zh-TW.rc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_TRADITIONAL
+
+IDD_MAINWINDOW DIALOGEX 0, 0, 320, 200
+STYLE DS_SETFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+EXSTYLE WS_EX_APPWINDOW
+CAPTION "虛擬光碟機控制台"
+FONT 9, "新細明體"
+BEGIN
+    PUSHBUTTON "OK", IDC_MAINOK, 211, 179, 50, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
```
[the remark this answers is not in the document]
Comment (njlyf2011) [reply modules/rosapps/applications/vcdcontroltool/lang/zh-TW.rc]: Sorry……It went missing ……

File: dll/win32/shell32/lang/zh-TW.rc [hunk 3]
```diff
@@ -565,26 +565,26 @@ CAPTION "自動播放"
...
+CAPTION "關閉 ReactOS"
 FONT 9, "新細明體"
 BEGIN
     ICON 8240, -1, 6, 6, 21, 20, SS_REALSIZECONTROL | WS_GROUP
-    LTEXT "What do you want the computer to do?", -1, 39, 7, 167, 10
+    LTEXT "您希望電腦做什麼?", -1, 39, 7, 167, 10
     COMBOBOX 8224, 39, 20, 165, 200, CBS_DROPDOWNLIST | WS_VSCROLL
-    LTEXT "Maintains your session, keeping the computer running on low power with data still in memory. The computer wakes up when you press a key or move the mouse.", 8225, 39, 40, 167, 37
+    LTEXT "維持工作階段，电脑數據仍然保持在 RAM 中。當你按下一個鍵或移動鼠標將喚醒這台電腦。", 8225, 39, 40, 167, 37
     DEFPUSHBUTTON "OK", 1, 7, 82, 60, 14, WS_GROUP
```
[the remark this answers is not in the document]
Comment (njlyf2011) [reply dll/win32/shell32/lang/zh-TW.rc]: Sorry, I don't understand the meaning ……

[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/shell32/lang/zh-TW.rc]: Can you translate 'OK' into Chinese?

**Comment by njlyf2011** [comment]
The word "memory" if direct translation into 「記憶體」will cause misunderstanding, so I translated it into "RAM"。

File: dll/win32/shell32/lang/zh-TW.rc (hunk 3 above)
[the remark this answers is not in the document]
Comment (njlyf2011) [reply dll/win32/shell32/lang/zh-TW.rc]: Ok, this is really my fault. I will fix the error after a while.

File: dll/win32/userenv/userenv.rc (hunk 1 above)
Comment (HBelusca) [reply dll/win32/userenv/userenv.rc]: Ah yes, there's a missing one in that file.

**Comment by njlyf2011** [comment]
Fixed.

File: dll/win32/shell32/lang/zh-TW.rc (hunk 3 above)
[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/win32/shell32/lang/zh-TW.rc]: dll/win32/shell32/lang/zh-TW.rc

You forgot this file's "OK".

**Comment by katahiromz** [comment]
dll/win32/shell32/lang/zh-TW.rc:620:18: warning: missing terminating " character [enabled by default]

**Comment by njlyf2011** [comment]
I am very sorry to have brought so much trouble to ReactOS Developers.
I hope this commit will not be wrong.

**Comment by HBelusca** [comment]
That's fine, PRs are for being reviewed :D

**Pull request merged**